text/0984-drop-safari-less-than-14.md

```diff
@@ -0,0 +1,95 @@
+---
+stage: accepted
+start-date: 2023-11-02T15:40:00.000Z 
+release-date: # In format YYYY-MM-DDT00:00:00.000Z
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
+  - learning
+  - steering
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/984
+project-link:
+suite: 
+---
+
+
+<!--- 
+Directions for above: 
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+suite: Leave as is
+-->
+
+# Drop support for Safari < v14.1 
+
+## Summary
```

Comment:
instead, mention:
- bugfixes / what we test against
- changes to how Apple supports and releases safari